src/components/feature-hints/feature-hint-dot.js

```diff
@@ -1,12 +1,11 @@
 import React, { useEffect, useState } from 'react';
 import { connect } from 'react-redux';
 
-import { featureHintsContent } from './feature-hints-content';
-
 import './feature-hints.scss';
 
 const FeatureHintDot = ({
   appState,
+  featureHintsContent,
```

Comment:
I am confused again because there are a lot of featureHints changes in this PR which seem out of scope and there's no information in the description so I don't know have much context of why they are in the PR.

Reply:
+1

Reply:
We want to highlight this new feature to the users, don't we? 

<img width="1717" alt="image" src="https://github.com/kedro-org/kedro-viz/assets/16869061/697da8e6-7088-4835-bd7b-3dd5820ac50c">

Did it not make sense when you checked out the branch and ran it locally?